web/src/redux/actions/board.action.ts

```diff
@@ -0,0 +1,11 @@
+import { createAction, props } from '@ngrx/store';
```

Comment:
redux directory can be removed and keep it in store.

Reply:
fixed